docs/SETUP_SIK.md

```diff
@@ -0,0 +1,23 @@
+# mLRS Documentation: Setup as SiK Telemetry Replacement #
+
+([back to main page](../README.md))
+
+This page describes how to set up an mLRS system to work as a bi-directional serial MAVLink telemetry link, independent on any radio, where mLRS' rc data capability is not desired and mLRS should only transmits and receives serial MAVLink data. This setup finds application in situations where traditionally, e.g., SiK telemetry uints would be used.
```

Comment:
How about 'This page describes how to use mLRS as a bi-directional MAVLink telemetry link similar to a SiK telemetry unit.  This setup doesn't require a radio and will only transmit and receive MAVLink data'

Reply:
done